Chat App/Chat App/ChatComponent/ChatView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  ChatView.swift
+//  ChatAppV2
+//
+//  Created by tornike on 26.04.23.
+//
+
+import UIKit
+
+final class ChatView: UIView {
+    
+    //MARK: variables
+    
+    private let textInputComponentView = TextInputComponentView()
+    private var messages = [
+        Message(text: "Hello", date: Date()),
+        Message(text: "How are you?", date: Date()),
+        Message(text: "I'm doing well, thanks!", date: Date())
+    ]
```

Comment:
Have not found any assignment to this array, so should we make it a constant maybe ?